website/templates/project/addon/widget.mako

```diff
@@ -1,5 +1,5 @@
-<div class="panel panel-default" name="${short_name}">
-
+% if complete or 'write' in user['permissions']:
```

Comment:
Nici has commented on the desired UI for contributors of the different permission levels here: https://openscience.atlassian.net/browse/OSF-4161 . 

I think this implements that behavior, but i'm not completely sure. @zachjanicki Can you please verify?